### PR TITLE
Override health records params during import-db

### PR DESCRIPTION
**Pull request by iulianpascalau**
- override health records params during import-db

### CODE REVIEW

File: cmd/node/main.go [hunk 1]
```diff
@@ -1575,6 +1575,14 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 
 func applyCompatibleConfigs(isInImportMode bool, importDbNoSigCheckFlag bool, log logger.Logger, config *config.Config, p2pConfig *config.P2PConfig) {
 	if isInImportMode {
+		defaultSecondsToCheckHealth := 300                  //5minutes
+		defaultDianoseMemoryLimit := 6 * 1024 * 1024 * 1024 // 6GB
```
Comment (BeniaminDrasovean) [inline cmd/node/main.go:1579]: Diagnose?

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by valentin-lup — DISMISSED** [review]
Used successfully for import DB on mainnet shard 1

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by valentin-lup — APPROVED** [review]
Successfully testnet with importDB on mainnet shard 1

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
image

**Pull request merged**